TypeCobol.Analysis.Test/DfaCallPgmReport.cs

```diff
@@ -54,5 +54,26 @@ public void InBulkCallPgm88SetReportTest()
                 TypeCobol.Test.TestUtils.compareLines(path, result, expected, output);
             }
         }
+
+        [TestMethod]
+        public void ProcCallPgmReportTest()
+        {
+            string path = Path.Combine(CfgTestUtils.Report, "ProcCallPgm.cbl");
+            var cfgs = ParseCompareDiagnostics<DfaBasicBlockInfo<VariableSymbol>>(path, CfgBuildingMode.WithDfa);
```

Comment:
You can use ParseCompareDiagnosticsWithDfa as a shortcut

Reply:
done in e9f226e